src/Engine/Util/Memory.hpp

```diff
@@ -2,7 +2,7 @@
 
 
 
-#define USE_RPMALLOC
+// #define USE_RPMALLOC
```

Comment:
Can you remove this? This is a per-user setting

Reply:
I'm the one who put this in, so I should take it out. Can it be defined in the cmake?